### PR TITLE
Clear Readme

### PR DESCRIPTION
**Pull request by Oxicode**

### CODE REVIEW

**Comment by sagikazarmark** [comment]
Thanks!

**Comment by Oxicode** [comment]
cc @mtdowling

**Comment by sagikazarmark** [comment]
Sorry, forgot to hit the merge button.

File: README.md [hunk 1]
```diff
@@ -21,19 +21,18 @@ trivial to integrate with web services.
...
-// 'application/json; charset=utf8'
-echo $res->getBody();
-// '{"id": 1420053, "name": "guzzle", ...}'
-
-// Send an asynchronous request.
+$request = $client->request('GET', 'https://api.github.com/repos/guzzle/guzzle');
+
+echo $request->getStatusCode(); # 200
+echo $request->getHeaderLine('content-type'); # 'application/json; charset=utf8'
+echo $request->getBody(); # '{"id": 1420053, "name": "guzzle", ...}'
```
Comment (gmponos) [inline README.md]: this does not make sense.. it's not a request.. but a response

Comment (Oxicode) [reply README.md]: Ok, I create pr #2251

**Pull request merged**